sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderClient.java

```diff
@@ -113,8 +123,13 @@ public InputStream getMapTileV2(
      * @return metadata for a tileset in the TileJSON format.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public MapTileset getMapTileset(TilesetID tilesetId) {
-        return this.serviceClient.getMapTileset(tilesetId);
+    public MapTilesetPrivate getMapTileset(TilesetID tilesetId) {
```

Comment:
no returning MapTilesetPrivate to the public. Please look at search sdk for guide